memory_stress.py

```diff
@@ -0,0 +1,18 @@
+"""Memory-stress a long-running CodeJail-using process."""
+
+from codejail import safe_exec
+
+GOBBLE_CHUNK = int(1e7)
+
+def main():
+    gobble = []
+    for i in xrange(int(1e7)):
```

Comment:
The fact that the range boundry and GOBBLE_CHUNK have the same value is unimportant?